vuln/core/44.json

```diff
@@ -9,5 +9,5 @@
   "affectedEnvironments": [
     "all"
   ],
-  "severity": "unknown"
+  "severity": "critical"
```

Comment:
Critical is not mentioned here: https://nodejs.org/en/blog/vulnerability/december-2017-security-releases

Reply:
correct - the blog doesnt mention any severity at all but it does mention 

> This vulnerability has been assigned CVE-2017-15896.

which is https://nvd.nist.gov/vuln/detail/CVE-2017-15896 `critical`

---

i feel like the blog and the `unknown` entry both indicate that at publish time we didn't know, and that's what this PR is addressing.

Should the blogs posts be amended too?